app/assets/stylesheets/style.scss

```diff
@@ -627,6 +618,16 @@ label:hover {
     transition: width .05s linear;
     -webkit-transform: translateZ(0) scale(1,1);
     z-index: 1000;
+
+    .fa {
+      position: relative;
```

Comment:
Properties should be ordered display, font-size, height, position, text-align, vertical-align, width